Automattic/newspack-popups
# feat: add concept of prompt styles and add large style

### All Submissions:






### Changes proposed in this Pull Request:

This PR introduces the concept of Styles to prompts. This is a way to ship design updates (e.g. a larger border!) without disrupting current users.

It's based on the Block Styles so users should be fairly familiar with it.

_Editor's Sidebar_
![](https://user-images.githubusercontent.com/177929/188931617-93e018d9-a5ab-427d-b505-3ab1681b428c.png)

_Large Border, front-end_
<img width="1512" alt="Screenshot 2022-09-07 at 17 41 36" src="https://user-images.githubusercontent.com/177929/188933747-f31d253f-dc61-46fa-be1a-ffbcf8404b2d.png">

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Create new prompt
3. Play with the different styles (and inline/overlay) check if they work as expected
4. Edit an old prompt (ideally one that had no border -- if not switch back to master to create one)
5. See if the styles also work

### Other information:
